docker/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
     /bin/bash ~/miniforge.sh -b -p /opt/conda && \
     rm ~/miniforge.sh
 
-ENV PATH /opt/conda/bin:$PATH
+ENV PATH=/opt/conda/bin:$PATH
```

Comment:
Wow - did they change the spec or are we just lucky that this has always worked before?

Reply:
They changed the spec.  I discovered this from "legacy" warnings being thrown by `gcov` when I was messing around in 24.04